src/stp/plays/PenaltyThemPrepare.cpp

```diff
@@ -28,23 +28,23 @@ PenaltyThemPrepare::PenaltyThemPrepare() : Play() {
 uint8_t PenaltyThemPrepare::score(world::World* world) noexcept { return 100; }
 
 void PenaltyThemPrepare::calculateInfoForRoles() noexcept {
-    auto width = field.getFieldWidth();
-    auto length = field.getFieldLength();
+    const auto xPosition = -4 * control_constants::ROBOT_RADIUS;
+    const auto yPosition = Constants::STD_TIMEOUT_TO_TOP() ? field.getFieldWidth() / 2.2 : -field.getFieldWidth() / 2.2;
```

Comment:
I tried to find if the STD_TIMEOUT_TO_TOP() ever changes, but as far as i can see it's always false. Did i miss something?

Reply:
nope gaat fout ik ga t proberen te fixen

Reply:
This used to be not a 'constant' but a value we set through the interface. You typically don't change it but it's useful during a match to be able to change the side somewhere.

Reply:
Was this fixed? As the constant is still used in the newer version